clang/lib/Sema/TreeTransform.h

```diff
@@ -11099,6 +11099,19 @@ OpenACCClause *TreeTransform<Derived>::TransformOpenACCClause(
     ParsedClause.setDefaultDetails(
         cast<OpenACCDefaultClause>(OldClause)->getDefaultClauseKind());
     break;
+  case OpenACCClauseKind::If: {
+    Expr *Cond = const_cast<Expr *>(
+        cast<OpenACCIfClause>(OldClause)->getConditionExpr());
```

Comment:
Can we add an assert that Cond is not null?
(I guess the logic here is that if the if clause is invalid, it will never be transformed?)

Reply:
If the condition is invalid, we shouldn't have constructed an 'if' clause in the first place.  That said, I think an assert makes sense here.